docs/_rpc/ns-miner.md

```diff
@@ -70,3 +70,13 @@ Sets the etherbase, where mining rewards will go.
 | Go      | `miner.SetEtherbase(common.Address) bool`                   |
 | Console | `miner.setEtherbase(address)`                               |
 | RPC     | `{"method": "miner_setEtherbase", "params": [address]}`     |
+
+### miner_setGasLimit
+
+Sets the gas limit the miner will target when minning. 
```

Comment:
```suggestion
Sets the gas limit the miner will target when mining. 
```

Reply:
We should document behaviour before and after london hf

Reply:
@MariusVanDerWijden you mean the fact that it needs to be 2x'ed after London to get the same throughput? If so, I can add a line about that.